quickstart/azure/tutorial-container-webserver.md

```diff
@@ -33,7 +33,10 @@ In this tutorial, we'll use JavaScript to deploy a simple webserver Container In
             image: "nginx",
             memory: 1,
             cpu: 1,
-            port: 80,
+            ports: [{
+                    port: 80,
+                    protocol: "TCP"
+            }],
```

Comment:
Curious - Did the original not work?  I thought both approaches were supported?

Reply:
It worked but output a warning about it being deprecated.

```
$ pulumi up
Previewing update (dev):

     Type                             Name                        Plan       Info
     pulumi:pulumi:Stack              stateplus-ts-webserver-dev
 +   └─ azure:containerservice:Group  nginx                       create     1 warning

Diagnostics:
  azure:containerservice:Group (nginx):
    warning: urn:pulumi:dev::stateplus-ts-webserver::azure:containerservice/group:Group::nginx verification warning: "container.0.port": [DEPRECATED] Deprecated in favor of `ports`

Resources:
    + 1 to create
    3 unchanged

...
```